src/OpenTelemetry.Exporter.ApplicationInsights/Implementation/TraceExporterHandler.cs

```diff
@@ -214,7 +214,7 @@ public async Task ExportAsync(IEnumerable<ISpanData> spanDataList)
 
                     // BUILDING resulting telemetry
                     OperationTelemetry result;
-                    if (resultKind == SpanKind.Client)
+                    if (resultKind == SpanKind.Client || resultKind == SpanKind.Consumer)
```

Comment:
Looking into https://opentracing.io/docs/best-practices/
producer is a span created to track sending message to the queue - this is definitely dependency. 
consumer is a span to track message processing and is a request from AppInsights perspective